lib/localeapp/cli/pull.rb

```diff
@@ -17,7 +19,11 @@ def execute
       def update_backend(response)
         @output.puts "Success!"
         @output.puts "Updating backend:"
-        Localeapp.updater.dump(Localeapp.load_yaml(response))
+        if @target_dir.nil?
```

Comment:
I would rewrite this as `unless @target_dir`. Or `if @target_dir` and
then reverse the actions.